System/Posix/Files.hsc

```diff
@@ -245,6 +245,8 @@ access name flags =
 -- | @getFileStatus path@ calls gets the @FileStatus@ information (user ID,
 -- size, access times, etc.) for the file @path@.
 --
+-- This will dereference symbolic links, if you want to get the status of a link file, use 'getSymbolicLinkStatus' instead.
```

Comment:
Could you please copy this comment to other incarnations of `getFileStatus` in this package?